pcsx2/Frontend/MetalHostDisplay.mm

```diff
@@ -293,10 +293,20 @@ static void OnMainThread(Fn&& fn)
 	ImGui::Render();
 	dev->RenderImGui(ImGui::GetDrawData());
 	dev->EndRenderPass();
+	bool use_present_drawable = false;
+	if (m_vsync_mode != VsyncMode::Off)
+		use_present_drawable = true;
+	if (@available(macOS 13, *)) // Before Ventura, presentDrawable acts like vsync is on when windowed
```

Comment:
I'm assuming the overhead from the `@available` bit is low? Not that it matters a ton, given it's called once per frame, but if not, could be worth caching it?

Reply:
Was a bit more than I expected (~100 instructions)
Cached (and added a env variable override because why not)